pkg/apis/helm.bitnami.com/v1/types.go

```diff
@@ -24,10 +25,17 @@ type HelmReleaseSpec struct {
 	ChartName string `json:"chartName,omitempty"`
 	// Version is the chart version
 	Version string `json:"version,omitempty"`
+	// Auth is the authentication
+	Auth HelmReleaseAuth `json:"auth,omitempty"`
 	// Values is a string containing (unparsed) YAML values
 	Values string `json:"values,omitempty"`
 }
 
+type HelmReleaseAuth struct {
+	// Header is header based Authorization
+	Header *corev1.EnvVarSource `json:"header,omitempty"`
```

Comment:
Perhaps a `*corev1.SecretKeySelector` ?  `EnvVarSource` has several other options that don't make sense in this context (like downardapi references).